### ISSUE TITLE
Unable to replicate Database limitation

### ISSUE THREAD

**Issue by Thrameos** [jpype-project/jpype#446]
When is not finding a bug a bad thing?  When the bug is in our limitation page.  

I tried to exercise the bug using an example from a sqlite tutorial.  No issues were found other than cygwin does not pass environment variables.

```
import os
import jpype
import jpype.imports

jpype.startJVM(jpype.getDefaultJVMPath(),'-Djava.io.tmpdir=%s'%os.environ['USERPROFILE'], classpath="sqlite-jdbc-3.27.2.1.jar")

import java.lang
from java.lang import Class
from java.util import Properties
from java.sql import Connection, DriverManager, SQLException

cls=Class.forName("org.sqlite.JDBC").newInstance();
connection = DriverManager.getConnection("jdbc:sqlite:testdb.db");
```

**Comment by marscher** [comment]
Can you reproduce it with the the bug-loaded 0.6.2 version?

**Comment by Thrameos** [comment]
I tried it with JPype 0.6.2 at one point and couldn't get it there either.  This is my second bite at the apple.. However, I believe this is just like the forName issue.  When calling a static class method that accesses the classloader, if called from within JPype it gets the system Boot ClassLoader (the one used to load java.lang.*) rather than the running SystemClassLoader.  The boot loader doesn't know anything about the jars that have been loaded, so it just returns ClassNotFound.  The problem with replicating it is that it likely is tied not just to our version but rather to the JVM and which path in the code the call hits.  Class forName always hits the bug, but I don't know the magic required to make DriverManager hit the bug.  

My plan is that given that this particular call structure (call a static method that accesses ClassLoader) is a really rare pattern, I would just add a JPype class customizer that forwards the call to a JPype internal call where I can execute the method with the given arguments.  As I am calling from within a Java class method the ClassLoader would be correct.  The limitation would still exist because I am just removing the common cases that hit the bug.

**Comment by Thrameos** [comment]
Fixed by #509
